tests/ctst/steps/pra.ts

```diff
@@ -274,6 +275,26 @@ Then('object {string} should {string} be {string} and have the storage class {st
         }
     });
 
+When('the DATA_ACCESSOR user tries to perform PutObject on {string} site', async function (this: Zenko, site: string) {
+    if (site === 'DR') {
+        Identity.useIdentity(IdentityEnum.ACCOUNT, `${Zenko.sites['source'].accountName}-replicated`);
+    } else {
+        Identity.useIdentity(IdentityEnum.ACCOUNT, Zenko.sites['source'].accountName);
+    }
+    this.resetCommand();
+    this.addToSaved('accountName', Zenko.sites['source'].accountName);
```

Comment:
shouldn't this be in the if/else above, so we use the -replicated one if we are using the data_accessor role from this account?
If we want to use this for the later tests, we should put this line at the end of the step

Reply:
The account name in zenkos is always the same, the `-replicated` is only used in the test to determine the account we are using, but in the end the account name is the same

Reply:
that said im not sure i understand this
```If we want to use this for the later tests, we should put this line at the end of the step```